doc/Overview/Reactive/in-apps.md

```diff
@@ -103,8 +103,7 @@ so you can enhance the UX of the pagination, like display a loading indicator wh
 
 
 ## Commands
-The generated bindable counterpart of a class will automatically re-expose public methods that have 0 or 1 parameter and an optional `CancellationToken` as `ICommand`.
-The parameter will be fulfilled using the `CommandParameter` property.
+The generated bindable counterpart of a class will automatically re-expose as `ICommand` public methods that are compatible (cf. below).
```

Comment:
Compatible with what ?

Reply:
Rules are explain "below", but there is no anchor that I can refer to :/